backend/editor/models.py

```diff
@@ -23,51 +23,51 @@ class Footer(Marginal):
 
 # Models for FastAPI
 
-class CommonParameters:
-    branch = "branch"
-    taxonomy_name = "taxonomy_name"
 
-class Entry:
-    default = []
-    title: str
-    description: str
+class CommonParameters(BaseModel):
+    branch: str = Query(description="Name of the branch")
+    taxonomy_name: str = Query(description="Name of the taxonomy")
+
+
+class Entry(CommonParameters):
+    entry: str = Query(description="Name of the entry")
+    id: str = Query(description="Id of the entry")
+
 
 class ImportFromGithubParameters(CommonParameters):
     pass
 
 
 class ImportFromGithubResponse(BaseModel):
-    status: bool
+    status: bool = Query(description="Return true or false value")

```

Comment:
The description does not add anything to the type, which is bool.

```suggestion
    status: bool = Query(description="True if import was successful")

```